Documentation/Contributing/documentation.md

```diff
@@ -0,0 +1,5 @@
+---
+title: Documentation
```

Comment:
Why do we need this page? Will more info be coming on this page later? Maybe we just leave this page out until there is more info. But maybe it would be sufficient to just have comments in the code or scripts where needed instead of having official docs on it.

Reply:
I have added some more info, but still would like to keep the page in to have a "documentation" for the documentation.

In the future we might add more information on writing docs and formatting guidelines.
What do you think?

Reply:
With more content it looks good, thanks